code/modules/mob/living/simple_animal/hostile/retaliate/pet.dm

```diff
@@ -0,0 +1,23 @@
+/mob/living/simple_animal/hostile/retaliate/araneus
```

Comment:
The one on Metastation's actually a subtype of /bat. It's because Araneus shouldn't have anything that spiders do have (poisonous bites, spider abilities, an enormous urge to kill people on sight).